clang/unittests/Basic/SourceManagerTest.cpp

```diff
@@ -453,6 +454,61 @@ TEST_F(SourceManagerTest, loadedSLocEntryIsInTheSameTranslationUnit) {
 
 #if defined(LLVM_ON_UNIX)
 
+TEST_F(SourceManagerTest, ResetsIncludeLocMap) {
```

Comment:
Suggestion: add a comment explaining what this aims to test at a more detailed level. Something like:
```cpp
// Check we clear include loc caches when resetting the source manager.
```